docs/api/shallow.md

```diff
@@ -47,9 +47,12 @@ describe('<MyComponent />', () => {
 1. `node` (`ReactElement`): The node to render
 2. `options` (`Object` [optional]):
   - `options.context`: (`Object` [optional]): Context to be passed into the component
-  - `options.disableLifecycleMethods`: (`Boolean` [optional]): If set to true, `componentDidMount`
-is not called on the component, and `componentDidUpdate` is not called after
-[`setProps`](ShallowWrapper/setProps.md) and [`setContext`](ShallowWrapper/setContext.md).
+  - `options.disableLifecycleMethods`: (`Boolean` [optional]): If set to true, most of the React
+lifecycle methods will never be executed: `componentDidMount`, `componentWillReceiveProps`,
+`shouldComponentUpdate`, `componentWillUpdate` and `componentDidUpdate`. Default to `false`.
+  - `options.lifecycleExperimental`: (`Boolean` [optional]): If set to true, the entire lifecycle
+(`componentDidMount` and `componentDidUpdate`) of the React component is called. The current default value
```

Comment:
Not really. If you do not specify any boolean setContext, setState and setProps will trigger  part of React lifecycle methods. If you disableLifecycleMethods nothing is executed whereas if you use lifecycleExperimental all the react lifecycle can be called.